AutoRest/Generators/CSharp/Azure.CSharp.Tests/Expected/AcceptanceTests/HeadExceptions/HeadExceptionOperations.cs

```diff
@@ -141,7 +141,12 @@ internal HeadExceptionOperations(AutoRestHeadExceptionTestService client)
             if ((int)_statusCode != 200)
             {
                 var ex = new CloudException(string.Format("Operation returned an invalid status code '{0}'", _statusCode));
-                _responseContent = await _httpResponse.Content.ReadAsStringAsync().ConfigureAwait(false);
+                if (_httpResponse.Content != null) {
```

Comment:
Just idle curiosity, but why not simplify statements like this into things like: 

```  csharp 
_httpResponse _httpResponse.Content != null ?  await _httpResponse.Content.ReadAsStringAsync().ConfigureAwait(false) : string.Empty; 
```

or better yet:

``` csharp
// would guarantee that we always get back a string, even if ReadAsStringAsync() returns null...
_httpResponse = await _httpResponse?.Content.ReadAsStringAsync().ConfigureAwait(false) ?? string.Empty; 
```

but, I'm funny that way :)